lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -76,6 +93,14 @@ def formatted_source
         def file_finished(_file, offenses)
           @offenses = offenses
         end
+
+        sig { returns(T::Boolean) }
+        def rubocop_version_supports_raise_cop_error?
+          RuboCop::Options.new.parse(["--raise-cop-error"])
+          true
+        rescue OptionParser::InvalidOption
+          false
+        end
```

Comment:
This implementation means that we will likely keep calling `parse` multiple times to detect the same unchanging feature. We should either memoize this value or do this operation at top level and assign the result to a constant to make it always run once. With memoization it would be like:
```suggestion
        sig { returns(T::Boolean) }
        def rubocop_version_supports_raise_cop_error?
          return @raises_cop_error if defined?(@raises_cop_error)
          
          @raises_cop_error = begin
            RuboCop::Options.new.parse(["--raise-cop-error"])
            true
          rescue OptionParser::InvalidOption
            false
          end
        end
```